packages/path_provider/lib/path_provider.dart

```diff
@@ -76,6 +76,18 @@ Future<Directory> getApplicationDocumentsDirectory() async {
 /// to access outside the app's sandbox.
 ///
 /// On Android this uses the `getExternalFilesDir(null)`.
+Future<Directory> getExternalFilesDirectory() async {
+  if (Platform.isIOS)
+    throw UnsupportedError("Functionality not available on iOS");
+  final String path =
+      await _channel.invokeMethod<String>('getExternalFilesDir');
+  if (path == null) {
+    return null;
+  }
+  return Directory(path);
+}
+
+
```

Comment:
It seems like you removed the Dartdoc from `getExternalStorageDirectory` and applied it to `getExternalFilesDirectory`. They  should both be documented.